core/domain/feedback_services_test.py

```diff
@@ -232,7 +232,52 @@ def test_get_all_threads_without_suggestion(self):
             with self.swap(utils, 'get_time_in_millisecs',
                            self._get_milliseconds):
                 threads = feedback_services.get_all_threads(
-                    self.EXP_ID2, False)
+                    self.EXP_ID1, False)
```

Comment:
Whether the existing tests are correct depends on what the author's intention was. Presumably, when the self.swap() is used, the author intends to test the behaviour of what lies "in between" FeedbackThreadModel.get_threads() and feedback_services.get_all_threads(); when it is not used, the author intends to test the behaviour of feedback_services.get_all_threads().

Having said that, using self.swap() does introduce some skew. I think it would be nicer to remove it, so please feel free to go ahead and do that if it isn't too hard.